lib/pages/Mainpage.dart

```diff
@@ -1,12 +1,22 @@
+import 'dart:math';
+
 import 'package:MobileOne/pages/bottom_bar.dart';
 import 'package:MobileOne/pages/lists.dart';
 import 'package:MobileOne/pages/loyalty_card.dart';
+import 'package:MobileOne/providers/loyalty_cards_provider.dart';
 import 'package:MobileOne/pages/profile.dart';
 import 'package:MobileOne/pages/share.dart';
-
+import 'package:MobileOne/services/user_service.dart';
+import 'package:barcode_scan/barcode_scan.dart';
 import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
 import 'package:permission_handler/permission_handler.dart';
 
+var uuidCard;
```

Comment:
MainPage should not hold card variables

Reply:
No, because floatting button need them. When we scan, we add loaylty card's data in database.